src/main/java/de/oliver/fancyholograms/storage/FlatFileHologramStorage.java

```diff
@@ -232,7 +233,9 @@ public static HologramData readHologram(String name, ConfigurationSection config
             final var billboardName = config.getString("billboard", DisplayHologramData.DEFAULT_BILLBOARD.name());
             final var textAlignmentName = config.getString("text_alignment", TextHologramData.DEFAULT_TEXT_ALIGNMENT.name());
             final var linkedNpc = config.getString("linkedNpc");
-            final var visibleByDefault = config.getBoolean("visible_by_default", DisplayHologramData.DEFAULT_IS_VISIBLE);
+            final var visibleByDefault = Optional.of(config.getString(
+                    "visible_by_default", DisplayHologramData.DEFAULT_IS_VISIBLE.toString())
+            ).map(Visibility::byString).orElse(DisplayHologramData.DEFAULT_IS_VISIBLE);
```

Comment:
It would probably be a good idea to add backwards compatibility and support the old boolean option for a while.

Reply:
You want to check booleans and convert into Predicate?